ui/StatusQ/src/StatusQ/Components/StatusQrCodeScanner.qml

```diff
@@ -30,16 +27,16 @@ Item {
     property bool highlightContentZone: false
     property bool highlightCaptureZone: false
 
-    readonly property alias camera: camera
-    readonly property size sourceSize: Qt.size(videoOutput.sourceRect.width, videoOutput.sourceRect.height)
-    readonly property size contentSize: Qt.size(videoOutput.contentRect.width, videoOutput.contentRect.height)
-    readonly property real sourceRatio: videoOutput.sourceRect.width / videoOutput.sourceRect.height
+    //readonly property alias camera: camera
```

Comment:
No longer needed/wanted?

Reply:
replaced by `cameraAvailable` boolean, we don't want expose `camera` object direct as it has different api in qt 5/6